charts/rasa-x/templates/rasa-x-ingress.yaml

```diff
@@ -0,0 +1,49 @@
+{{- if and .Values.ingress.enabled (not .Values.nginx.enabled) -}}
+{{- $fullName := include "rasa-x.fullname" . -}}
+{{- $rasaxHost := include "rasa-x.host" . -}}
+{{- if semverCompare ">=1.14-0" .Capabilities.KubeVersion.GitVersion -}}
+apiVersion: networking.k8s.io/v1beta1
+{{- else -}}
+apiVersion: extensions/v1beta1
+{{- end }}
+kind: Ingress
+metadata:
+  name: {{ $fullName }}
+  labels:
+    {{- include "rasa-x.labels" . | nindent 4 }}
+  annotations:
+    nginx.ingress.kubernetes.io/proxy-body-size: "0"
+    nginx.ingress.kubernetes.io/proxy-read-timeout: "600"
+    nginx.ingress.kubernetes.io/proxy-send-timeout: "600"
+    nginx.ingress.kubernetes.io/configuration-snippet: |
+      if ($arg_environment = "") {
+          rewrite ^/api/chat /core/webhooks/rasa/webhook last;
+      }
+      if ($arg_environment = "production") {
+          rewrite ^/api/chat /core/webhooks/rasa/webhook last;
+      }
+  {{- with .Values.ingress.annotations }}
+    {{- toYaml . | nindent 4 }}
+  {{- end }}
+spec:
+{{- if .Values.ingress.tls }}
+  tls:
+  {{- range .Values.ingress.tls }}
+    - hosts:
+      {{- range .hosts }}
+        - {{ . | quote }}
+      {{- end }}
+      secretName: {{ .secretName }}
+  {{- end }}
+{{- end }}
+  rules:
+  {{- range .Values.ingress.hosts }}
+  - host: {{ .host | quote }}
+    http:
+      paths:
+      - backend:
+          serviceName: {{ $rasaxHost }}
```

Comment:
I think we need to treat the `/ws` path differently to make websockets work. At least I required that in our docker-compose setup.

Reply:
The same here, all additional configuration that is here https://github.com/RasaHQ/rasa-x/blob/master/docker/nginx/rasax.nginx is the default for nginx-ingress

Reply:
Ok! Did you do a test deploy to check if the websocket connections work?

Reply:
I did, and I tested :)

```
$ websocat -v ws://<ingress-adress>/api/ws
[INFO  websocat::lints] Auto-inserting the line mode
[INFO  websocat::ws_client_peer] get_ws_client_peer
[INFO  websocat::ws_client_peer] Connected to ws
```

Reply:
Thanks for `websocat` 👍 Didn't know it